CSharp/JobPrep/packages.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Azure.Batch" version="1.3.0" targetFramework="net45" />
+  <package id="Microsoft.Data.Edm" version="5.6.0" targetFramework="net45" />
+  <package id="Microsoft.Data.OData" version="5.6.0" targetFramework="net45" />
+  <package id="Microsoft.Data.Services.Client" version="5.6.0" targetFramework="net45" />
+  <package id="Microsoft.WindowsAzure.ConfigurationManager" version="1.8.0.0" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="6.0.3" targetFramework="net45" />
```

Comment:
which nuget page? I just traced this back to the 4.0.0 storage library and it shows Json 5.0.6 as its requirement.